res/values/strings.xml

```diff
@@ -594,6 +594,12 @@
     <string name="systemmsg_ephemeral_timer_week">Disappearing messages timer set to 1 week.</string>
     <string name="systemmsg_ephemeral_timer_four_weeks">Disappearing messages timer set to 4 weeks.</string>
 
+    <!-- Translators: Protection disabled/enabled messages will be displayed together with a big "shield" and may be suffixed by systemmsg_action_by_user or systemmsg_action_by_me -->
+    <string name="systemmsg_chat_protection_disabled">Chat protection disabled.</string>
+    <string name="systemmsg_chat_protection_enabled">Chat protection enabled.</string>
```

Comment:
```suggestion
    <string name="systemmsg_chat_protection_enabled">Chat protection on.</string>
```

Reply:
see comment  above.

Reply:
In my head this could also mean some sort of +voice mode. "Chat now protected" sounds less like that to me.
Is there a short way to explain how?